### PR TITLE
add tag filter

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**

### CODE REVIEW

File: patches/d2l-tag-filter.patch [hunk 1]
```diff
@@ -0,0 +1,18391 @@
...
+-bower_components-1.x
+ 
+ # DS_Store
+ .DS_Store
++
++node_modules
+diff --git a/bower.json b/bower.json
+deleted file mode 100644
+index c465f6b..0000000
+--- a/bower.json
```
Comment (awikkerink) [inline patches/d2l-tag-filter.patch]: These shouldn't be included in the patch. Only changes after the `toPolymer3.sh` script is run

Comment (jstefaniuk-d2l) [reply patches/d2l-tag-filter.patch]: Oh shoot, good call

**Comment by jstefaniuk-d2l** [comment]
Fixed it for real this time 😄
